src/hearings/resolvers/panel-details.resolver.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable } from '@angular/core';
+import { ActivatedRouteSnapshot, Resolve } from '@angular/router';
+import { select, Store } from '@ngrx/store';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { HearingRequestMainModel } from '../models/hearingRequestMain.model';
+import { HearingRequestStateData } from '../models/hearingRequestStateData.model';
+import { JudicialUserModel } from '../models/judicialUser.model';
+import { JudicialRefDataService } from '../services/judicial-ref-data.service';
+import { LovRefDataService } from '../services/lov-ref-data.service';
+import * as fromHearingStore from '../store';
+
+@Injectable({
+  providedIn: 'root',
+})
+
+export class PanelDetailsResolver implements Resolve<JudicialUserModel[]> {
```

Comment:
it shouldn't be called PanelDetailsResolver here, if it's also used by the judgeDetailsResolver.

Reply:
Also I think @Kartheek-S has created this resolver.